File: Python/1046 Game Time.py
i, f = map( int, input( ).split( ))

if f>i:
    print('O JOGO DUROU {} HORA(S)'.format(f-i))
else:
    print('O JOGO DUROU {} HORA(S)'.format(24+f-i))

# Time: 0.016s

File: 1. Begginers/1046-Game_Time.cpp
#include <iostream>
using namespace std;

int a, b, hours;

main(){
    cin >> a>> b;
    hours = b - a;
    if (hours <= 0){
        hours += 24;
    }

    cout<< "O JOGO DUROU "<< hours<< " HORA(S)"<< endl;

}
